lib/easypost/beta.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require 'easypost/beta/beta_referral'
-
 module EasyPost::Beta
 end
+
+require_relative 'beta/referral'
```

Comment:
Ah, the ordering here is important? Meaning it needs to be required after defining the module? If so, this was probably the missing secret sauce from our initial attempts.

cc @jchen293

Reply:
That would fitr.